CMakeLists.txt

```diff
@@ -543,8 +545,14 @@ else()
   message(STATUS "Using static fmt from Externals")
   add_subdirectory(Externals/fmt EXCLUDE_FROM_ALL)
 endif()
-add_subdirectory(Externals/glslang)
 add_subdirectory(Externals/imgui)
+if(APPLE)
+	set(ENABLE_VULKAN 0)
+endif()
```

Comment:
Support for macOS and Vulkan via MoltenVK was added after this PR was made (Nov. 2019, see 673f1963a068fb7d5cd8af9bb209c7633950e8f8) so the two `!defined(__APPLE__)`s below should be able to be safely removed! I tested this removal locally to confirm things build sanely.